module1/lessons/hashes.md

```diff
@@ -1,11 +1,17 @@
 ---
 title: Hashes
+subheading: lesson is Hashes 
```

Comment:
Remove this line, it shows up on the actual lesson plan. Or change it to something like: `Ruby data structure`, or just omit.